scapy/layers/bluetooth4LE.py

```diff
@@ -128,6 +129,33 @@ def getfield(self, pkt, s):
         return s[5:], self.m2i(pkt, struct.unpack(self.fmt, s[:5] + b"\x00\x00\x00")[0])  # noqa: E501
 
 
+class BTLEFeatureField(FlagsField):
+    def __init__(self, name, default):
+        FlagsField.__init__(self, name, default, -64,
+                            ['le_encryption',
+                             'conn_par_req_proc',
+                             'ext_reject_ind',
+                             'slave_init_feat_exch',
+                             'le_ping',
+                             'le_data_len_ext',
+                             'll_privacy',
+                             'ext_scan_filter',
+                             'le_2m_phy',
+                             'tx_mod_idx',
+                             'rx_mod_idx',
+                             'le_coded_phy',
+                             'le_ext_adv',
+                             'le_periodic_adv',
+                             'ch_sel_alg',
+                             'le_pwr_class'])
```

Comment:
I feel like the formatting could be made a bit better.

```suggestion
        super(BTLEFeatureField, self).__init__(
            name, default, -64,
            ['le_encryption',
             'conn_par_req_proc',
             'ext_reject_ind',
             'slave_init_feat_exch',
             'le_ping',
             'le_data_len_ext',
             'll_privacy',
             'ext_scan_filter',
             'le_2m_phy',
             'tx_mod_idx',
             'rx_mod_idx',
             'le_coded_phy',
             'le_ext_adv',
             'le_periodic_adv',
             'ch_sel_alg',
             'le_pwr_class']
        )
```